src/scripts/angry-tock.js

```diff
@@ -4,14 +4,14 @@ const scheduler = require("node-schedule");
 const {
   dates: { getCurrentWorkWeek },
   slack: { sendDirectMessage },
-  tock: { get18FTockSlackUsers, get18FTockTruants },
+  tock: { get18FTockSlackUsers, get18FUsersWhoHaveNotTocked },
```

Comment:
suggestion (non-blocking): In case you want it, shorter variable/function names like `get18FLateTockers` or `lateTockingSlackUsers` popped into mind. I don't feel strongly about this at all, just an idea.